pipeline-steps/deploy.groovy

```diff
@@ -0,0 +1,19 @@
+def deploy(){
+  common.conditionalStep(
+    step_name: "Deploy",
+    step: {
```

Comment:
Could we also have the option of deploying using the deploy.sh script? From a QE perspective, we'll want to also test that deploy.sh is also working properly in addition to the individual playbooks. We could also make it a separate pipeline-step if that would be more straightforward (like a deploy-playbooks and a deploy-script kind of business).